pkg/vbmh/machines.go

```diff
@@ -289,7 +295,12 @@ func (ml *MachineList) countScheduledAndTobeScheduled(nodeRole airshipv1.VMRoles
 		if readyScheduled {
 			logger.Info("BMH host is not yet marked as ready to be scheduled, marking it as ready to be scheduled")
 			// Add it to the list.
-			ml.Machines[bmh.ObjectMeta.Name] = NewMachine(bmh, nodeRole, Scheduled)
+			m, err := NewMachine(bmh, nodeRole, Scheduled)
+			if err != nil {
+				logger.Info("Skipping BMH host as it did not meet creation requirements", "error", err.Error())
```

Comment:
From the user's perspective, they have already created machines. I would recommend changing this to "did not meet scheduling requirements"

Reply:
done